rasa/core/policies/ted_policy.py

```diff
@@ -677,25 +698,25 @@ def _emebed_dialogue(
             dialogue_transformed = tf.expand_dims(
                 self._last_token(dialogue_transformed, sequence_lengths), 1
             )
-            mask = self._last_token(mask, sequence_lengths)
+            mask = tf.expand_dims(self._last_token(mask, sequence_lengths), 1)
```

Comment:
why do we need to `expand_dims` now?

Reply:
because we do that for `dialogue_transformed` above. Then both max history and full dialogue featurizers have sequence dimension

Reply:
all of these millions of `expand_dims` and `squeeze` need to be cleaned though, let's add it to further TODOs list